src/app/process.go

```diff
@@ -151,16 +160,23 @@ func (p *Process) run() int {
 
 func (p *Process) getProcessStarter() func() error {
 	return func() error {
-		p.command = command.BuildCommandShellArg(p.shellConfig, p.getCommand())
+		p.command = command.BuildCommand(
+			p.procConf.Executable,
+			append(p.procConf.Args, p.extraArgs...),
```

Comment:
Could you give an example of the final command you’d like executed?

Reply:
```yaml
processes:
  ls:
    command: "ls"
```
```shell
process-compose run ls -- /tmp
```
`run ls` will ignore the extraArg `/tmp`
It worked well with `entrypoint` instead of `command`.

Reply:
This prints: `args: 0 1 2`

```bash
bash -c 'echo args: $0 $1 $2' 0 1 2 
```

Reply:
Try:
```shell
bash -c "ls" /tmp
```
vs.
```shell
bash -c "ls /tmp"
```